h/groups/schemas.py

```diff
@@ -23,7 +23,7 @@ class GroupSchema(CSRFSchema):
             min=GROUP_NAME_MIN_LENGTH,
             max=GROUP_NAME_MAX_LENGTH),
         widget=deform.widget.TextInputWidget(
-            autofocus=True,
```

Comment:
Indeed - WTF moment. That's a mistake. The change to disable the outline is in the SCSS. I'll fix and re-test.